woocommerce/woocommerce-gateway-dummy
# Upgraded to node 20

### Description 

This PR upgrades the required node version to v20 and updates dev dependencies. 

closes #35 
closes #32 

### Testing steps

1. Check out this branch. 
2. [Build](https://github.com/woocommerce/woocommerce-gateway-dummy?tab=readme-ov-file#building-instructions) the plugin.
3. Activate Dummy Gateway.
4. Complete a test order -- ensure that Dummy Gateway shows up in the checkout page.
